python/cudf/cudf/tests/test_applymap.py

```diff
@@ -13,7 +13,7 @@
 @pytest.mark.parametrize(
     "nelem,masked", list(product([2, 10, 100, 1000], [True, False]))
 )
-def test_applymap_round(nelem, masked):
+def test_apply_round(nelem, masked):
```

Comment:
This basically tests `floor` which we might (?) need to add to `apply`. We should do that and then migrate this test to `test_udf_masked_ops.py` which contains most of the tests for `apply`.

Reply:
Ah, this is actually just `math.floor` from above which is [supported and tested](https://github.com/rapidsai/cudf/blob/f5aa39d640b1eb5a9999234814e3e2092ceda6a5/python/cudf/cudf/core/udf/_ops.py#L31). As such this is just a fancy test for floor plus some arithmetic and is covered elsewhere. I'd be in support of removing this.

Reply:
Awesome. This is also tested in https://github.com/rapidsai/cudf/blob/3b2b12fd60c39fb040596e40813d30aa245f510b/python/cudf/cudf/tests/test_seriesmap.py#L56.